metrics/storage/fio_job.sh

```diff
@@ -72,24 +77,49 @@ EOF
 
 # Create the job configuration for FIO
 # this job will run inside of the container.
-function create_fio_job()
+# This job sets up the image - creates the test file
+function create_fio_prep_job()
 {
 	FIO_JOB="$(cat << EOF
```

Comment:
Just found a couple of uses of `FIO_JOB`, which got deprecated when I split it into RUN and PREP phases ... think I got bit by a rebase.... new push coming.